edit.ml

```diff
@@ -110,61 +110,16 @@ let split_edge instrs pc label pc' =
   new_instrs, pc_map
 
 let replace_uses_in_instruction old_name new_name instr : instruction =
```

Comment:
no, I thought of eg. returning the map object or even push down this method (since it has only one caller I think)